enterprise/models.py

```diff
@@ -94,21 +109,33 @@ class EnterpriseCustomerUser(TimeStampedModel):
     """
 
     enterprise_customer = models.ForeignKey(EnterpriseCustomer, blank=False, null=False)
-    user_id = models.PositiveIntegerField(null=False, blank=False)
```

Comment:
+1 for @bradenmacdonald's suggestion. It's more robust and consistent with how things are handled elsewhere in edx-platform.